packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -34,6 +34,13 @@
 		&[data-field-id="actions"] {
 			text-align: right;
 		}
+
+		&:focus,
+		&:focus-visible,
+		&:focus-within {
+			background-color: #f0f0f0;
+			outline: none;
+		}
```

Comment:
This gives focused cells, or cells containing focused elements, more prominence in the table. Could do with some design input here.
// @WordPress/gutenberg-design

Reply:
From the video demo you shared the gray seems darker in that demo than #f0f0f0 (btw you could use `$gray-100` there if you like). Was the video recorded earlier or is it just a visual illusion?

In any case my instinct is that if we are also showing the focus-ring, changing the background at the same time is likely going to be a bit noisy. It also might confuse if we end up adding a full-row background-color change on mouse hover. I'll defer to Jay, though, if he chimes in.

Reply:
It'd be good to understand the full requirements here before committing to a design. 

I assume a focus style is required for cells that only contain data like the Status column in the video?

As you navigate focus transfers immediately to the first child of each cell where possible. In these cases do parent cells still need to be visually highlighted?

If the answer to these questions is yes then perhaps we need different styles for focus and focus-within.

Reply:
@andrewhayward Given that this table should and will be applied to many contexts, I feel like the outline should remain for focus. That particular gray might blend with a page background somewhere.

Reply:
Sorry, I'll provide a little more context for clarity.

The request for feedback is more around how we ensure that focal context is clearly communicated, rather than on the way I specifically did it here.

The current background colour is somewhat arbitrary (I think I copied it from the unfocused search field), so could be anything; frankly, the cells don't have to have a background colour at all, as long as focus is clear.

Interactive cell content receives focus as normal, but the cells themselves receive focus when they contain no interactive content. I used a background here to maintain consistency between these two different states, but it's not strictly necessary. To @jeffgolenski's point, the cells themselves could have a focus ring, and it wouldn't be a problem.

Essentially, the problem of how best to communicate the focal context is not straightforward, and doesn't necessarily map onto our existing patterns (though it could).

Reply:
We tend to use rings to denote focus throughout the Editor so I'd be inclined to try repeating that pattern here.